lib/api/core/entryPoints/lb.js

```diff
@@ -0,0 +1,71 @@
+var RequestObject = require('kuzzle-common-objects').Models.requestObject;
+
+/**
+ * Manage communication with the load balancer
+ * @param kuzzle
+ */
+function Lb (kuzzle) {
+  this.kuzzle = kuzzle;
+  this.channels = {};
+}
+
+Lb.prototype.init = function () {
+  this.kuzzle.services.list.lbBroker.listen('request', onRequest.bind(this));
+  this.kuzzle.services.list.lbBroker.listen('connection', onConnection.bind(this));
+  this.kuzzle.services.list.lbBroker.listen('disconnect', onDisconnect.bind(this));
+  this.kuzzle.services.list.lbBroker.listen('error', onDisconnect.bind(this));
+};
+
+Lb.prototype.joinChannel = function (data) {
+  this.kuzzle.services.list.lbBroker.send('joinChannel', data);
+};
+
+Lb.prototype.leaveChannel = function (data) {
+  this.kuzzle.services.list.lbBroker.send('leaveChannel', data);
+};
+
+Lb.prototype.notify = function (data) {
+  if (this.isDummy) {
+    return false;
+  }
+
+  this.kuzzle.services.list.lbBroker.send('notify', data);
+};
+
+Lb.prototype.broadcast = function (data) {
+  if (this.isDummy) {
+    return false;
+  }
+
+  this.kuzzle.services.list.lbBroker.send('broadcast', data);
+};
+
+function onRequest (data) {
+  var requestObject;
+
+  if (data.request && data.context && data.context.connection && data.context.connection.type) {
+    requestObject = new RequestObject(data.request, data.request.data, data.context.connection.type);
+
+    this.kuzzle.funnel.execute(requestObject, data.context, (error, responseObject) => {
```

Comment:
why do you call directly funnel.execute() instead of calling routerController.execute() method (cf. https://github.com/kuzzleio/kuzzle/blob/KUZ-552-lb-communication/lib/api/controllers/routerController.js#L78 ) ?

Are the checks implemented in the routerController.execute() method unnecessary ? if so, I guess we should simply remove the routerController.execute() method, as plugin protocols are now moved to LB...